FnCAS/fncas/base.h

```diff
@@ -64,7 +64,23 @@ enum class type_t : uint8_t { variable, value, operation, function };
 enum class operation_t : uint8_t { add, subtract, multiply, divide, end };
 enum class function_t : uint8_t { sqr, sqrt, exp, log, sin, cos, tan, asin, acos, atan, unit_step, ramp, end };
```

Comment:
Auch :-)

Thx for the catch. `NodeType`, `MathOperation`, `MathFunction` in 187ab5e.